src/net_processing.cpp

```diff
@@ -6270,6 +6270,10 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
 
                 // Determine transactions to relay
                 if (fSendTrickle) {
+                    if (m_txreconciliation && m_txreconciliation->IsPeerRegistered(pto->GetId())) {
+                        // Make transactions added to the reconciliation set during the last interval available
+                        m_txreconciliation->ReadyDelayedTransactions(pto->GetId());
```

Comment:
In baac72581b1c99a41ab16a00a3f25fe938eb0610: "This prevents adversarial nodes from proving
our reconciliation set by spamming reconciliation requests": Couldn't we just check the last time the node did a reconciliation request and simply ignore the request?